thetower/dtower/thetower/urls.py

```diff
@@ -1,40 +0,0 @@
-import os
-
-from django.contrib import admin
-from django.urls import path
-
-from dtower.sus.views import SusView
-from dtower.tourney_results.views import plaintext_results, plaintext_results__champ, results_per_tourney, results_per_user
-
-base_patterns = [path("admin/", admin.site.urls)]
-
-
-urlpatterns = base_patterns
-
-league_text_patterns = [
-    path("<str:league>/text/<str:tourney_date>/", plaintext_results),
-    path("<str:league>/text/", plaintext_results),
-]
```

Comment:
I meant text patterns can go, not entire api. The json patters are so that skye doesn't kill us with requests.

Reply:
Fixed.